packages/akane-next/src/app/(main)/(map)/(bottomSheet)/(pageRoot)/create/page.tsx

```diff
@@ -23,64 +31,65 @@ function StoryCreateTop({
     }[];
 }) {
     return (
-        <Box>
-            <Text textStyle="2xl">謎解き作成</Text>
-            <VStack>
-                <Field.Root>
-                    <Field.Label>タイトル</Field.Label>
-                    <Field.HelperText>ストーリータイトルを入力してください</Field.HelperText>
-                    <Field.ErrorText>タイトルが入力されていません</Field.ErrorText>
-                    <Field.RequiredIndicator />
-                    <Input placeholder="ストーリータイトル" />
-                </Field.Root>
-                <Field.Root>
-                    <Field.Label>画像</Field.Label>
-                    <Field.HelperText>ストーリーの画像を選択してください</Field.HelperText>
-                    <Field.ErrorText>画像が選択されていません</Field.ErrorText>
-                    <Field.RequiredIndicator />
-                    <Input type="file" />
-                </Field.Root>
-                <Field.Root>
-                    <Field.Label>所要時間(参考)</Field.Label>
-                    <Field.HelperText>ストーリーの所要時間を入力してください</Field.HelperText>
-                    <Field.ErrorText>所要時間が入力されていません</Field.ErrorText>
-                    <Field.RequiredIndicator />
-                    <Input placeholder="分" />
-                </Field.Root>
-                <VStack>
-                    {/* ここに問い${index}を表示 */}
-                    {nazos.map((nazo) => (
-                        <HStack
-                            key={nazo.id}
-                            borderWidth="1px"
-                            borderRadius="lg"
-                            p={2}
-                            onClick={() => handleCreateQuestion()}
-                        >
-                            <Text>問い {nazo.id}</Text>
-                        </HStack>
-                    ))}
-                    <Button onClick={() => handleCreateQuestion()}>問題を追加する</Button>
-                </VStack>
-                <VStack>
-                    <Button
-                        colorScheme="green"
-                    >
-                        投稿
-                    </Button>
-                    <Button
-                        colorScheme="blue"
-                    >
-                        下書き保存
-                    </Button>
-                    <Button
-                        colorScheme="red"
-                    >
-                        作成をキャンセル
-                    </Button>
-                </VStack>
-            </VStack>
-        </Box>
+      <Box p={4}>
+      <Text textStyle="2xl" mb={4} >謎解き作成</Text>
+      <VStack>
+          {/* タイトル */}
+          <Field.Root required mb={4}>
+              <Field.Label>タイトル <Field.RequiredIndicator /></Field.Label>
+              <Field.ErrorText>タイトルが入力されていません</Field.ErrorText>
+              <Input placeholder="ストーリータイトル" />
+          </Field.Root>
+
+          {/* 画像アップロード */}
+          <Field.Root required mb={4}>
+              <Field.Label>画像 </Field.Label>
```

Comment:
[must] 必須項目のラベルには` <Field.RequiredIndicator />`をつけるべきです。
```suggestion
              <Field.Label>
                 <Field.RequiredIndicator />
                画像
               </Field.Label>
```